certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -395,6 +395,99 @@
         },
         "deprecated": false,
         "x-ms-visibility": "advanced"
+      },
+      "put": {
+        "tags":[
+          "DocuSign"
+        ],
+        "summary": "Update envelope prefill tabs",
+        "description": "Update value for a list of prefill tabs",
```

Comment:
Update **values** for a list of prefill tabs

Reply:
Done